src/Legacy/Numbers/Words/Locale/Tr.php

```diff
@@ -76,7 +76,7 @@ class Tr extends Words
         'BWP' => [['Botswana pulası'], ['thebe']],
         'BYR' => [['Belarus rublesi'], ['kopiejka']],
         'CAD' => [['Kanada doları'], ['sent']],
-        'CHF' => [['İsveç frankı'], ['rapp']],
+        'CHF' => [['İsviçre frankı'], ['rapp']],
```

Comment:
```suggestion
        'CHF' => [['İsviçre frangı'], ['rapp']],
```

It's written as `İSVİÇRE FRANGI` in [the Central Bank of the Republic of Türkiye's website](https://www.tcmb.gov.tr/kurlar/today.xml#:~:text=%C4%B0SV%C4%B0%C3%87RE%20FRANGI).

Reply:
> It's written as `İSVİÇRE FRANGI` in [the Central Bank of the Republic of Türkiye's website](https://www.tcmb.gov.tr/kurlar/today.xml#:~:text=%C4%B0SV%C4%B0%C3%87RE%20FRANGI).

Thanks :)